src/sentry/services/hybrid_cloud/project/impl.py

```diff
@@ -72,3 +76,32 @@ def serialize_many(
             user=as_user,
             serializer=ProjectSerializer(),
         )
+
+    def create_project_for_organization(
+        self,
+        *,
+        organization_id: int,
+        project_name: str,
+        platform: str,
+        user_id: int,
+        add_org_default_team: Optional[bool] = False,
```

Comment:
On the fence about this being a bool vs a team ID. This saves me from having to do an org/team lookup via RPC in our signup code, but makes this method less reusable 🤔

Reply:
Are there scenarios where we'd want to encourage the use of `create_project_for_organization` via RPC in the future? I can't think of any but 🤷

Reply:
The major one I can think of would be our Fly IO provisioning which creates multiple projects, but that may need something more specific that can idempotently bulk create them. CC: @sentaur-athena

Reply:
Couldn't we revisit this and add bulk project creation when the need arises?